docs/server.md

```diff
@@ -0,0 +1,40 @@
+# Playing with the API
+
+If you want to experience API usage in a test environment,
+in addition to the API documentation, you can download and use a NodeJS (express)
+HTTP server which implements the APIs based on the OpenAPI specifications.
+
+## Downloading the test environment
+
+Start cloning the GitHub repository:
+
+`git clone http://github.com/teamdigitale/cittadinanza-digitale`
+`cd cittadinanza-digitale`
+`npm install`
+
+## Starting the mock server
+
+Type
+
+`npm run serve`
```

Comment:
also here a multi line block, it's good practice to use single backtick code blocks only inline